src/components/Pages/ManageWiki/Features/EntityMapping.vue

```diff
@@ -24,20 +24,43 @@
         Some tools assume items with special meanings under certain identifiers. 
 		Here, you can map a item on your Wikibase instance to an item on Wikidata.
       </v-card-text>
-	  <ItemsTable/>
+	  <ItemsTable ref="itemTable" @save="onSave" />
 	</v-card>
 </div>
 </template>
 
 <script>
-import PropertiesTable from './PropertiesTable.vue';
-import ItemsTable from './ItemsTable.vue';
+import PropertiesTable from './PropertiesTable.vue'
+import ItemsTable from './ItemsTable.vue'
 
 export default {
   name: 'EntityMapping',
   components: { PropertiesTable, ItemsTable },
-  props: [],
+  props: ['wikiId'],
   methods: {
+    toMapping (entities) {
+      const mappings = []
+      for (var i = 0; i < entities.length; i++) {
+        const mapping = entities[i]
+        mappings[mapping.local] = mapping.wikidata
+      }
+      return mappings
```

Comment:
Manifest actually splits the array in properties and items, not everything all together.
example from my `LocalSettings.php`:
```
$wgWbManifestWikidataEntityMapping = [
    'properties' => [
        "P31" => 'P1'
    ],
    'items' => [
        'Q42' => 'Q1'
    ]
];
```
We should reflect it here, either with some check of what is passed or maybe split it in two functions, whatever sounds easier.

Reply:
done! https://github.com/wbstack/api/pull/35